r/R/development_helpers.R

```diff
@@ -1,9 +1,26 @@
+#' Generates request for worker task
+#'
+#' @param task
+#'
+#' @return
+#' @export
+#'
+#' @examples
 generate_request <- function(task) {
     res = switch(task,
-           "expression" = list(body=list(genes=list("gzma")))
+           "expression" = list(body=list(genes=list("gzma"))),
+           "markerHeatmap" = list(body=list(nGenes=5,type="louvain",config=list(resolution=0.5)))
```

Comment:
am I right that after Martin's changes on the UI and the API, these settings won't be hardcoded here, but they will be coming from the front end?

Reply:
These functions are for development purposes only, they won't actually be used besides for internal use.